en/_posts/2021-10-06-019.md

```diff
@@ -1,14 +1,22 @@
 ---
 layout: post
-title: '#019 What is a remote?'
-image: "https://res.cloudinary.com/jesstemporal/image/upload/f_auto/v1642878597/gitfichas/en/019/thumbnail_hzoxpz.jpg"
+pretitle: What is a
+title: remote
+subtitle:
+concept: true
+parts:
+  - part1: is a cloud repository\nfor example on Github or GitLab
+  - part2: every time you make a clone\na repository you have at least one\nremote - the origin
```

Comment:
```suggestion
  - part2: every time you clone a repository you have at\nleast one remote - the origin
```
to make this part the same as in the card image:
![Image](https://github.com/user-attachments/assets/0be21ec5-0e88-4605-a68b-880d1fa6d33f)